app/views/entities/index.html.erb

```diff
@@ -0,0 +1,299 @@
+    content: "";
+    position: absolute;
+    top: 5px;
+    bottom: 5px;
+    left: 5px;
+    right: 5px;
+    border: 1px solid white; 
+    pointer-events: none; 
+}
+
+
+.icon-list {
+    display: flex;
+    flex-wrap: wrap;
+    gap: 20px;
+    justify-content: center;
+}
+
+.icon-item {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    margin: 5px;
+    padding: 10px;
+    border: 1px solid #e0e0e0;
+    border-radius: 5px;
+    transition: background-color 0.3s;
+}
+
+.icon-item:hover {
+    background-color: #f2f2f2;
+}
+
+.form-control {
+    width: 100%;
+    font-size: 16px;
+    padding: 12px 20px;
+    border: none;
+    border-radius: 0;
+    box-shadow: none;
+    background-color: #f3f3f3;
+    box-sizing: border-box;
+}
+
+.form-group {
+    margin-bottom: 20px;
+}
+
+.form-group.mt-4 {
+    text-align: center;
+    position: fixed;
+    right: 160px;
+}
+
+.table {
+    border-collapse: collapse;
+    margin-top: 20px;
+    width: 100%;
+    box-shadow: 0 5px 10px rgba(0,0,0,0.1);
+    font-family: 'Proxima Nova', sans-serif;
+}
+
+.table th, .table td {
+    padding: 15px 20px;
+}
+
+.table thead {
+    background-color: #f4f4f4;
+    color: #lightgray;
+}
+
+.table tbody tr:hover {
+    background-color: #f4f4f4;
+}
+
+h3.text-center {
+    display: flex;
+    justify-content: center;
+    font-size: 24px;
+    padding: 10px 0;
+    background-color: #f7f7f7;
+    border-bottom: 1px solid #e0e0e0;
+    border-top: 1px solid #e0e0e0;
+    font-family: 'Proxima Nova', sans-serif;
+}
+
+.btn-secondary {
+    background-color: #fff;
+    border: none;
+    color: #adb5bd;
+}
+
+</style>
+<div class="container">
+    <nav class="navbar navbar-expand-lg" style="background-color: #3778c2; padding: 20px 0">
+        <div class="login-nav">
+            <span class="navbar-text">
+                Transactions: <%= @group.name %>
+            </span>
+            <% if user_signed_in? %>
+                <%= button_to "Log Out", destroy_user_session_path, method: :delete, class: "login-btn" %>
+            <% end %>
+        </div>
+    </nav>
+
+    <h3 class="text-center mt-4">Total: $ <%= @total_amount %></h3>
+
+    <div class="table-responsive mt-4">
+        <table class="table">
+            <thead>
+                <tr>
+                    <th>Name</th>
+                    <th>Amount</th>
+                </tr>
+            </thead>
+            <tbody>
+                <% @entities.each do |entity| %>
+                    <tr>
+                        <td><%= entity.name %></td>
+                        <td><%= entity.amount %></td>
+                    </tr>
+                <% end %>
+            </tbody>
+        </table>
+    </div>
+
+    <!-- Botones después de la tabla -->
+    <div class="horizontal-center">
+        <%= link_to 'ADD NEW TRANSACTION', new_group_entity_path(@group), class: 'btn btn-primary mt-1' %>
+        <%= link_to 'Back', root_path, class: "btn btn-secondary" %>
```

Comment:
hi @TedLivist, thanks for the review.

I added it that way because it's that way that I believe it fixes the best way for my application needs, it says that it's just a guideline, not to replicate the design.

#### Design
  - You should follow these [design guidelines](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=), including:
  -- Colors.
  -- Typography: font face, size and weight.
  -- Layout: composition and space between elements.